bowtie/__init__.py

```diff
@@ -10,3 +10,8 @@
 GITHUB = URL.parse("https://github.com/")
 ORG = GITHUB / "bowtie-json-schema"
 REPO = ORG / "bowtie"
+
+GITHUB_API = URL.parse("https://api.github.com/")
```

Comment:
These can go in the CLI module I think, they're unlikely to be used anywhere outside it (whereas the rest of the stuff here are used in multiple places).

Also it's fine to just have one variable, I don't think anything but `CONTAINER_PACKAGES_API` is likely to be used anywhere either.

Reply:
> These can go in the CLI module I think

Meaning should I put them in `_core.py` itself? since I have added the `get_versions` method on `Implementation` itself.

Reply:
Yes, there is fine too.